src/modules/drawing/DrawingModule.vue

```diff
@@ -88,7 +89,16 @@ watch(availableIconSets, () => {
         const feature = drawingLayer.getSource()?.getFeatureById(featureId)?.get('editableFeature')
         if (feature?.icon) {
             const iconArgs = parseIconUrl(feature.icon.imageURL)
-            const icon = getIcon(iconArgs, null /*iconStyle*/, availableIconSets.value)
+            const icon = getIcon(iconArgs, null /*iconStyle*/, availableIconSets.value, () => {
+                store.dispatch('addWarnings', {
+                    warnings: [
+                        new WarningMessage('kml_icon_set_not_found', {
+                            iconSetName: iconArgs.set,
+                        }),
+                    ],
+                    dispatcher: 'kmlUtils.js',
```

Comment:
Wrong dispatcher use the one on top of the file

Reply:
But this one is dispatched from `kmlUtils.js` as a callback. I follow the same approach as in https://github.com/geoadmin/web-mapviewer/blob/develop/src/modules/map/components/openlayers/OpenLayersKMLLayer.vue#L86-L112

or should I change that one too?

Reply:
Yes and no, the callback is defined outside of kmlUtils.js and the caller is the file that call the helper function in kmlUtils.js.
I think for clarity it might make more sense to keep the file that contains the definition as dispatcher. This dispatcher can be useful for debugging so you will first look in kmlUtils.js and it will be then hard to find the dispatch in this file.